src/Shared/ErrorPage/README.md

```diff
@@ -0,0 +1,14 @@
+# Error Page
+
+This folder is shared among multiple projects. The `ErrorPage.Designer.cs` and `ErrorPageModel.cs` files are used to render this view. The `ErrorPage.Designer.cs` file is generated by rendering `src/Views/ErrorPage.cshtml` using the [RazorPageGenerator](https://github.com/aspnet/AspNetCore-Tooling/tree/master/src/Razor/src/RazorPageGenerator) tool.
```

Comment:
Nice! Always nice to make the grass greener for the next person to work with the error page!

Reply:
Agreed. Even on platforms without powershell, you can look at the script to see how the generator is supposed to be invoked.